integrationtest/org/daisy/dotify/formatter/test/resource-files/marker-ref/marker-reference-page-first-workaround-input.obfl

```diff
@@ -2,18 +2,18 @@
 <obfl xmlns="http://www.daisy.org/ns/2011/obfl" version="2011-1" xml:lang="und" hyphenate="false">
    <meta>
       <dc:title xmlns:dc="http://purl.org/dc/elements/1.1/">marker-reference-page-first-workaround</dc:title>
-      <dc:description xmlns:dc="http://purl.org/dc/elements/1.1/">Tests (hypothetical) marker-reference type that corresponds with 'page-first' in CSS. This type is not yet directly supported in OBFL. As a workaround I do a forward page search (left corner in this test), and if that does not return a value, a backward sequence search (right corner). Tested with markers at the beginning of the sequence, in the middle of a page, at a soft page break within a block, at a soft page break between two blocks, and at a hard page break.</dc:description>
+      <dc:description xmlns:dc="http://purl.org/dc/elements/1.1/">Tests (hypothetical) marker-reference type that corresponds with 'page-first' in CSS. This type is not yet directly supported in OBFL. As a workaround I do a forward page search, and if that does not return a value, a backward sequence search. Tested with markers at the beginning of the sequence, in the middle of a page, at a soft page break within a block, at a soft page break between two blocks, and at a hard page break.</dc:description>
```

Comment:
This file has "workaround" in its name. It that still actual?

Reply:
Changed the name.